lib/feature/class-feature.php

```diff
@@ -30,8 +30,70 @@ class Feature {
 	 */
 	public static $feature_ids = [];
 
-	public static function is_enabled( $feature ) {
-		return static::is_enabled_by_percentage( $feature );
+	/**
+	 * Holds feature slug and then, key of environments with bool value to enable E.g.
+	 * // Enable feature for non-production sites
+	 * // 'feature-flag' => [ 'non-production' => true ],
+	 *
+	 * @var array
+	 */
+	public static $feature_envs = [];
```

Comment:
I think this needs future clarification? Looking at the code below it allows to do both `non-production` and `specific_environment_slug`